### PR TITLE
 Portfolio: Add a media queries  to displays the website correctly on desktop screens

### PR DESCRIPTION
**Pull request by Benawi**
## Features Added:

- Media queries used in CSS file to make desktop version of the website
- For some elements animations implemented
- I try to stick the implimantaion to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: css/stylesDesktop.css [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+  .card-work {
+    align-items: center;
+    justify-content: space-between;
+    min-height: 474px;
+    width: 91%;
+    border-radius: 8px;
+    border: 1px solid #d0d9d4;
+    background-color: #ebf0ee;
+    margin-bottom: 24px;
+  }
```
Comment (muneebulrehman) [inline css/stylesDesktop.css:184]: Your work cards are not matching the Figma template of your choosing. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: css/stylesDesktop.css [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  .about-list.flex-container-column {
+    flex-direction: row;
+    width: 81vw;
+    max-width: 1200px;
+    justify-content: space-around;
+    margin-top: 28px;
+    margin-bottom: 144px;
+  }
+
+  .about-list-item {
```
Comment (muneebulrehman) [inline css/stylesDesktop.css:203]: As per the Figma template, each `.about-list-item` should have a `border: 1px solid #C1C7D0;` which is missing in your case. Kindly include this border in your design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Benawi  , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
## CHANGES  ♻️
- Cards are matched with the Figma template of I choose [the templates in Figma](  https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=2ZThPFk85QKmYAaJ-0)
- To .about-list-item css class added  a border: 1px solid #C1C7D0;  As per requested on the Figma template
- In the footer section horzonatal bar shape disabled for desktop version

File: css/stylesDesktop.css [hunk 3]
```diff
@@ -0,0 +1,305 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+
+/* RESIZING FOR DESKTOP VERSION */
+@media only screen and (min-width: 1100px) {
+  /* Disabled Elements */
+
+  /* Enabled Elements */
```
Comment (iambenkis) [inline css/stylesDesktop.css]: - [ ]  [**OPTIONAL**]Good job setting the desktop version of your portfolio, however the value for `min-width` property should be in range of `768px` or wider . Noticed that you did set the `min-width` at `1100px` and wider. Kindly update that before you merge the PR 😄 

Capture 1

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**